WebApp/autoreduce_webapp/templates/accessibility_statement.html

```diff
@@ -0,0 +1,123 @@
+        </p>
+
+        <h3>How accessible this website is</h3>
+        <p>
+            The Auto-reduction Development Team is actively making an effort to improve accessibility. We know some parts of this website are not fully accessible:
+        </p>
+        <ul>
+            <li>Some text boxes do not have WCAG-Compliant contrast with their background</li>
+            <li>Some form controls do not have associated labels</li>
+            <li>Some adjacent text links are for the same resource</li>
+        </ul>
+
+        <h3>Feedback and contact information</h3>
+        <p>
+            If you need information on this website in a different format like accessible PDF, large print, easy read, audio recording or braille please email:
+            <a href="mailto:{{ support_email }}">{{ support_email }}</a>. We’ll consider your request and get back to you as soon as possible.
+        </p>
+
+        <h3>Reporting accessibility problems with this website</h3>
+        <p>
+            We’re always looking to improve the accessibility of this website. If you find any problems not listed on this page or think we’re not meeting accessibility requirements, contact:
+        </p>
+        <ul>
+            <li>The Auto-reduction Development Team: <a href="mailto:{{ support_email }}">{{ support_email }}</a></li>
+            <li>STFC: <a href="mailto:enquiries@stfc.ac.uk">enquiries@stfc.ac.uk</a></li>
+        </ul>
+
+        <h3>Enforcement procedure</h3>
+        <p>
+            The Equality and Human Rights Commission (EHRC) is responsible for enforcing the Public Sector Bodies (Websites and Mobile Applications) (No. 2) Accessibility Regulations 2018 (the 'accessibility regulations'). If you're not happy with how we respond to your complaint, contact the
+            <a rel="external"
+               href="https://www.equalityadvisoryservice.com/">Equality Advisory and Support Service (EASS)</a>
+        </p>
+
+        <del><h3>Contacting us by phone or visiting us in person</h3></del>
+
+        <h3>Technical information about this website’s accessibility</h3>
+        <p>
+            STFC is committed to making its website accessible, in accordance with the Public Sector Bodies (Websites and Mobile Applications) (No. 2) Accessibility Regulations 2018.
+        </p>
+
+        <h4>Compliance status</h4>
+        <p>
+            This website is partially compliant with the <a rel="external"
+                                                            href="https://www.w3.org/TR/WCAG21/">Web Content Accessibility Guidelines version 2.1</a> AA standard, due to the non-compliances listed below.
+        </p>
+
+        <h3>Non-accessible content</h3>
+        <p>
+            <i>The content listed below is non-accessible for the following reasons.</i>
+        </p>
+
+        <h4>Non-compliance with the accessibility regulations</h4>
```

Comment:
This needs to mention the plots - they are not accessible because they're generated dynamically in the reduction so putting relevant alt text on the images is a challenge and it's unlikely we'll do it